src/support_code_library_builder/index.js

```diff
@@ -40,13 +41,13 @@ export class SupportCodeLibraryBuilder {
   }
 
   defineStep(pattern, options, code) {
-    const stepDefinition = buildStepDefinition({
+    const stepDefinitionStruct = buildStepDefinitionStruct({
       pattern,
       options,
       code,
       cwd: this.cwd,
     })
-    this.options.stepDefinitions.push(stepDefinition)
+    this.options.stepDefinitions.push(stepDefinitionStruct)
```

Comment:
Can we store these at `this.options.stepDefinitionConfigs` and then after built it is stored at `this.options.stepDefinitions` and the configs property is removed

Reply:
and done